lib/panel-utils.js

```diff
@@ -324,6 +318,36 @@ function messageHandler(opts) {
   }
 }
 
+// Opens the video in the original tab, if it is still open, else in a new tab.
+function sendToTab(panel, opts) {
+  const pageUrl = getPageUrl(opts.domain, opts.id, opts.time);
+
+  if (pageUrl) {
+    const tabs = require('sdk/tabs');
+    let targetTab = null;
+    for (let tab of tabs) { // eslint-disable-line prefer-const
+      if (tab.id === opts.tabId) {
+        targetTab = tab;
+        break;
+      }
+    }
+
+    const targetVideoId = getVideoId(targetTab.url);
+    const pageVideoId = getVideoId(pageUrl);
+    if (targetTab !== null && pageVideoId && targetVideoId === pageVideoId) {
+      targetTab.url = pageUrl;
```

Comment:
ahh I see, the original intention was to simple activate the already opened tab. You make a good point about continuing playback though. Somehow I wasn't taking that into account. I'll think about it a bit and play with it over the weekend and see what feels best. Thanks!